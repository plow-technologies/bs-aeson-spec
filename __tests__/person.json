{ "name" : "Javier"
, "age" : 45
}
