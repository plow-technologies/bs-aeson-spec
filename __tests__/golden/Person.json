{
  "age": 24,
  "name": "Joan"
}
